phpunit.xml

```diff
@@ -13,7 +13,7 @@
 -->
 <phpunit bootstrap="vendor/weitzman/drupal-test-traits/src/bootstrap.php">
   <php>
-    <env name="DTT_BASE_URL" value="http://prisoner-content-hub-backend:8080"/>
+    <env name="DTT_BASE_URL" value="http://drupal:8080"/>
     <env name="DTT_API_URL" value="http://chrome:9222"/>
     <!-- <env name="DTT_MINK_DRIVER_ARGS" value='["chrome", { "chromeOptions" : { "w3c": false } }, "http://localhost:4444/wd/hub"]'/> -->
```

Comment:
Do we need the commented out code?

Reply:
The xml file comes from Drupal (inc the commented code), so it’s there just to keep it as close to the original as possible.

also this PR isn’t modifying the commented part. If we wanted to removed it I think it should be in a separate PR